index.html

```diff
@@ -155,10 +155,13 @@
 <a>Decentralized identifiers</a> (DIDs) are a new type of identifier that
 enables verifiable, decentralized digital identity. A <a>DID</a> identifies any
 subject (e.g., a person, organization, thing, data model, abstract entity, etc.)
-that the controller of the <a>DID</a> decides that it identifies. These new
-identifiers are designed to enable the controller of a <a>DID</a> to prove
-control over it and to be implemented independently of any centralized registry,
-identity provider, or certificate authority. <a>DID</a>s are URLs that associate
+that the controller of the <a>DID</a> decides that it identifies.
+
+DIDs are implemented independently of any centralized registry, identity provider,
```

Comment:
This isn't correct (and was the topic of much debate in the WG). DIDs enable non-centralized registries to be used for this purpose but DO NOT require them. As strange as it sounds, you can have a centralized Decentralized Identifier system (it's a corner case, but one that Microsoft argued for).

What are you attempting to say in this PR? Perhaps those of us that have been doing this for 5+ years can help you articulate it in a way that achieves your goal while not undoing years of debate over the sentences in the front matter of the document.

Reply:
> What are you attempting to say in this PR?

I was attempting to make the sentence easier to read, not subvert the, "... hard fought language [without which] the WG was unable to get started" :rofl:

OK, the original reads, "These new identifiers are designed to enable the controller of a DID to prove control over it and to be implemented independently of any centralized registry, identity provider, or certificate authority."

Lets break it down:

* "These new identifiers are..." - New to who? Why stress that they are new? New as of when? What makes them new? I'd rather just say how they are different / special and let the reader understand that they are new rather than telling them they are new (which raises all the above questions).

* "... designed to enable ..." - Fine.

* "... the controller of a DID to prove control over it ..." - Sounds tautological and is jargon heavy. I know what you mean, but it's clunky and hard to understand. An alternative could be, "In contrast to federated identifiers, the controller of a DID alone can prove control over it without permission from any other centralized registry, identity provider, or certificate authority" ... But I'd rather just relegate the jargon to the second part of the sentence.

* "... and to be implemented independently of any centralized registry, identity provider, or certificate authority." - To me this is the key point of the sentence. I get the nuance now, it isn't a defining feature, just something that they are designed to allow...

Putting it all together, how about...
* "In contrast to typical, federated identifiers, DIDs have been designed so that they may be implemented independently of any centralized registry, identity provider or certificate authority. Specifically, DIDs enable the controller of an identity alone to prove control over it without permission from any other party".

If not, I give up. I didn't mean to wander into this minefield. I'm not totally tone deaf to the concerns being voiced here, but I didn't expect this small 'copy edit' to be so controversial! Hope you understand :-)